nixos/modules/services/databases/mysql.nix

```
@@ -590,10 +595,10 @@ in
             # If MariaDB is used in an Galera cluster, we have to check if the sync is done,
             # or it will fail to init the database while joining, so we get in an broken non recoverable state
             # so we wait until we have an synced state
-            if ${cfg.package}/bin/mysql -u ${superUser} -N -e "SHOW VARIABLES LIKE 'wsrep_on'" 2>/dev/null | ${lib.getExe' pkgs.gnugrep "grep"} -q 'ON'; then
+            if ${cfg.package}/bin/${binaryPrefix} -u ${superUser} -N -e "SHOW VARIABLES LIKE 'wsrep_on'" 2>/dev/null | ${lib.getExe' pkgs.gnugrep "grep"} -q 'ON'; then
               echo "Galera cluster detected, waiting for node to be synced..."
               while true; do
-                STATE=$(${cfg.package}/bin/mysql -u ${superUser} -N -e "SHOW STATUS LIKE 'wsrep_local_state_comment'" | ${lib.getExe' pkgs.gawk "awk"} '{print $2}')
+                STATE=$(${cfg.package}/bin/${binaryPrefix} -u ${superUser} -N -e "SHOW STATUS LIKE 'wsrep_local_state_comment'" | ${lib.getExe' pkgs.gawk "awk"} '{print $2}')
```

```suggestion
                STATE=$(${lib.getExe' cfg.package binaryPrefix} -u ${superUser} -N -e "SHOW STATUS LIKE 'wsrep_local_state_comment'" | ${lib.getExe' pkgs.gawk "awk"} '{print $2}')
```
You get the gist. Please do the same for the remaining invocations as well

---

Should I edit only the ones which I've modified, or should I apply this change everywhere (e.g. `${cfg.package}/bin/galera_recovery`)?